src/IW/Sync/Update.hs

```diff
@@ -28,13 +31,68 @@ fetchAndUpsertRepos
        , WithError m
        )
     => m ()
-fetchAndUpsertRepos = do
-    gitHubRepos <- fetchAllHaskellRepos
+syncRepos = do
+    today <- liftIO getToday
+    sync syncReposByDate firstHaskellRepoCreated today 15 1
```

Comment:
I feel like this function can be improved a lot. But at the same time I feel that if we spend a lot of time improving this function, we will never get to the rest of the project :laughing: From the top of my head:

1. We can use `newtype`s here
2. For now you do GitHub query for every batch of issues, but you can fetch all issues first and then insert all at once. Though, not sure that it will be faster.

But I think that the current implementation is good enough.

Reply:
Yes! I agree. I think I've finally realized a better way to do this.